app/move/fields/move-not-agreed-instruction.js

```diff
@@ -0,0 +1,12 @@
+const moveNotAgreedInstruction = {
+  skip: true,
+  component: 'raw',
+  html:
+    '<p class="govuk-body">Complex cases must be discussed and agreed with the receiving prison.</p><p class="govuk-body">This includes:<p><ul class="govuk-list govuk-list--bullet"><li>Segregated prisoners</li><li>Self harm/prisoners on ACCT</li><li>Mental health issues</li><li>Integrated Drug Treatment System</li></ul>',
```

Comment:
Not particularly a big fan of having all this markup in a field config but if that's what we need to build...